### PR TITLE
Citation file

### PR DESCRIPTION
**Pull request by gvwilson**

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    orcid: ""
+  - family-names: "Steinmacher"
+    given-names: "Igor"
+    orcid: ""
+  - family-names: "Ford"
+    given-names: "Denae"
+    orcid: ""
+  - family-names: "Averick"
+    given-names: "Mara"
+    orcid: ""
```
Comment (batpigandme) [inline CITATION.cff]:
```suggestion
    orcid: "0000-0001-9659-6192"
```

**Review by batpigandme — COMMENTED** [review]
Thanks, Greg!

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,50 @@
+cff-version: 1.2.0
+message: "Please cite this paper as shown."
+authors:
+- family-names: "Sholler"
+  given-names: "Dan"
+  orcid: ""
+- family-names: "Steinmacher"
+  given-names: "Igor"
+  orcid: ""
```
Comment (igorsteinmacher) [inline CITATION.cff]:
```suggestion
  orcid: "0000-0002-0612-5790"
```

Comment (igorsteinmacher) [reply CITATION.cff]: Thanks for adding that Greg

File: CITATION.cff (hunk 1 above)
Comment (gvwilson) [reply CITATION.cff]: added with domain

**Pull request merged**